ios-base/Extensions/FontExtension.swift

```diff
@@ -29,11 +32,17 @@ extension UIFont {
     return UIFont(descriptor: descriptor, size: pointSize)
   }
   
-  static func font(withName name: String, size: CGFloat) -> UIFont {
-    let size = Font.PointSize.proportional(to: (.screen6_5Inch,
-                                                size)).value()
+  static func font(withName name: String = "", size: CGFloat) -> UIFont {
+    let minSize = size - 1
+    let maxSize = size + 2
+
+    let proportionalSize = Font.PointSize
+      .proportional(to: (.screen4_7Inch, size)).value()
+
+    let finalSize = max(min(proportionalSize, maxSize ), minSize)
```

Comment:
please revert these changes. The issue with bigger fonts is not related to the ios-base, it is a known issue with RSFontSizes and the Device dependency being updated. We need to update the dependency and then update the pod version of RSFontSizes.

Reply:
shall we leave a TODO until the RSFintSizes issue is fixed?